src/main/java/org/folio/service/PostgresFunctionExecutionService.java

```diff
@@ -28,4 +29,20 @@ public Future<Void> runBudgetEncumbrancesRolloverScript(LedgerFiscalYearRollover
         });
     return promise.future();
   }
+
+  public Future<Void> dropTable(String tableName, boolean isTemporary, DBClient client) {
```

Comment:
why is there a need to drop a table dynamically?

Reply:
We need to drop temporary tables after rollover here: https://github.com/folio-org/mod-finance-storage/pull/306/files#diff-2c0fd1819ea9b05997fb9dfe3acd62faf650ca3e3a6372119a1690dbfff76ed7R194

Reply:
In general, we should allow the database schema to be stable. If the table name is always the same, it would make sense to just truncate the table. If the table name is always different, it makes sense to drop the table but probably should have a different approach so that database schema is stable. No changes necessary at this point.